### DISCUSSION TITLE
[v4 beta][angular]Update to v4-beta, then tailwindcss not work

### DISCUSSION THREAD

**Discussion by JavaProgrammerLB** [tailwindlabs/tailwindcss#15079] [Help]
1. I update tailwindcss in my angular project follow the [document](https://tailwindcss.com/docs/v4-beta#upgrading-from-v3)
```
npx @tailwindcss/upgrade@next
```
2. after that `<h1 class="bg-green-500 underline">hello, world</h1>` code not show with green background and underline
![Xnip2024-11-22_11-41-25](https://github.com/user-attachments/assets/15c34539-cb57-4429-97e3-2f90f4f369f4)

3. my code is:https://github.com/JavaProgrammerLB/cssPractiseV2/

4. I update tailwindcss from v3 to v4-beta in [this commit](https://github.com/JavaProgrammerLB/cssPractiseV2/commit/59532afb51a76a1350abb0fb9c10df96b365c5cb)

**Comment by philipp-spiess** [comment]
Hey! Is it possible that Angular runs the Tailwind compiler with an unexpected current working directory? 🤔 

You can try to customize the source paths in your CSS file: https://tailwindcss.com/docs/v4-beta#configuring-source-detection 

```css
@import "tailwindcss" source(".");
```

**Reply by JavaProgrammerLB** [reply]
copy that,I will try your solution    late

**Comment by JavaProgrammerLB** [comment]
`@import "tailwindcss" source(".");` or `@import "tailwindcss" source("../src");` not work, I will try study more to fix this or if you can give me some suggestion
<img width="931" alt="image" src="https://github.com/user-attachments/assets/549050db-dc88-47a2-88b6-b196ca6d9fdd">

**Comment by dotdotcommadot** [comment]
Did this ever get resolved?
I'm having the same issue with Angular 19 and Tailwind 4.0.1

**Reply by JavaProgrammerLB** [reply]
> Did this ever get resolved?
> 
> I'm having the same issue with Angular 19 and Tailwind 4.0.1

hi, my issue not fixed,I still use tailwindcss 3 in my project
